aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -955,6 +961,89 @@ void qtopk_kernel(Tensor& values,
   });
 }
 
+template <bool ReluFused>
+void q_batch_norm_kernel(
+    int64_t N,
+    int64_t C,
+    int64_t HxW,
+    const int64_t in_zero_point,
+    const int64_t out_zero_point,
+    const uint8_t* X,
+    const float* alpha,
+    const float* beta,
+    uint8_t* Y) {
+#if defined(__AVX2__) && defined(__FMA__)
+
+  constexpr int kVLen = 8;
+  const int outer_size = N * HxW;
+  using Vec = Vec256<quint8>;
+  for (int i = 0; i < outer_size; ++i) {
```

Comment:
I think we might be able to simplify this further, since--correct me if i'm wrong--this is essentially just an elementwise access pattern where we broadcast `alpha` and `beta` to all [N, H, W]. It might be pretty simple to just use TensorIterator instead of handling the loop nest explicitly. I can elaborate a bit more later

Reply:
Blocked by #33166